src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -342,15 +342,26 @@ object ErrorAggregator {
     }
 
     if (docType == "crash") {
-      parse(CrashPing(message), dimensions, statsSchema)
+      val crashPing = CrashPing(message)
+      if (crashPing.meta.normalizedBuildId.isEmpty) {
```

Comment:
hmm, this repetition of `normalizedBuildId.isEmpty` is bothering me. Along with your refactoring work in the other PR we should include some sort of `isProper` or `isSanitized` function to the `Ping` trait.

Reply:
Yeah, I admit it's not ideal - some clear encapsulation of validation logic (and calling it like this) should be done.